pcsa_GHN/models.py

```diff
@@ -12,14 +10,22 @@
 
 class Contact(models.Model):
     office_name = models.CharField(max_length=200)
-    contact_number = models.BigIntegerField()
+    contact_number = models.CharField(
+        max_length=20,
```

Comment:
@Tahirih-Natang correct me if I am wrong, but is not the `E.164` digit limit set to max 15, any reason why our max length is set to 20?